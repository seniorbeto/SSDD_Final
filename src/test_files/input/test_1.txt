register beto
connect beto
list_users
quit
