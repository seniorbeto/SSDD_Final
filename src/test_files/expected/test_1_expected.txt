c> c> REGISTER OK
c> c> CONNECT OK
c> c> LIST_USERS OK
	USER0: beto	127.0.0.1	55705
c>
+++ FINISHED +++